lib/vrl/stdlib/src/append.rs

```diff
@@ -1,4 +1,4 @@
-use vrl::prelude::*;
+use crate::prelude::*;
```

Comment:
How has this change been made possible? I'm not seeing anywhere where `prelude` has been pulled into the  stdlib crate..

Reply:
Moved it to `stdlib/mod.rs`: https://github.com/vectordotdev/vector/blob/28cc0d5036733a79deff62ed15840d4ce50cf643/lib/vrl/stdlib/src/lib.rs#L747-L760

Reply:
As discussed offline with @JeanMertz, we'll introduce a new `vrl` crate that includes `vrl_core` and all the other vrl related crates.

However, I'm not sure we can share this particular piece, since that would mean having a cyclic dependency on `vrl` -> `stdlib` -> `vrl`. We can add an equivalent prelude to `vrl` to be accessible from the outside, though.